### PR TITLE
rough draft of cams

### PR DESCRIPTION
**Pull request by jscho13**
Lemme know once either of you guys read through this and I'll hit merge.

### CODE REVIEW

File: app/views/welcome/gg.haml [hunk 1]
```diff
@@ -2,9 +2,6 @@
   %h1 TightFit
   %p fitting porn into your tight schedule
 
-  - 5.times do
-    %video{ controls: true, width: "320", height: "240" }
-      %source{ src: @videos.sample.link, type: "video/mp4" }
```
Comment (jscho13) [inline app/views/welcome/gg.haml:7]: I removed this cause it was causing the app to crash. Not sure if I should add it back in.

Comment (laudickson) [reply app/views/welcome/gg.haml:7]: Ya leave it out, probably something to do with heroku

File: app/assets/stylesheets/base/_cams.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
+
+.cam-model-container {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 1em 0 0 0;
+}
+
+.cam-model {
+  display: inline-block;
+  flex-grow: 1;
```
Comment (laudickson) [inline app/assets/stylesheets/base/_cams.scss]: If you want to use `flex-grow` you have to have `display: flex;` in the above line

File: config/environments/development.rb [hunk 3]
```diff
@@ -6,8 +6,9 @@
   # since you don't have to restart the web server when you make code changes.
   config.cache_classes = false
 
-  # Do not eager load code on boot.
-  config.eager_load = false
+  # https://stackoverflow.com/questions/38198668/rails-5-load-lib-files-in-production
+  # The default is false. We turn it to on so we can use load files in /lib
+  config.eager_load = true
```
Comment (laudickson) [inline config/environments/development.rb:11]: What does `eager_load` do?

File: db/schema.rb [hunk 4]
```diff
@@ -15,7 +15,7 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "videos", force: :cascade do |t|
+  create_table "video_link", force: :cascade do |t|
```
Comment (laudickson) [inline db/schema.rb]: oh you changed it back

Comment (laudickson) [reply db/schema.rb]: y?

File: config/environments/development.rb (hunk 3 above)
Comment (laudickson) [reply config/environments/development.rb:11]: Allows for `/lib` usage

**Pull request merged**